assets/shaders/darkness_post_processing.wgsl

```diff
@@ -28,7 +28,7 @@ fn fragment(in: FullscreenVertexOutput) -> @location(0) vec4<f32> {
     let widthf32 = vec2<f32>(f32(width.x), f32(width.y));
 
     let pos = in.position.xy;
-    let lantern_pos = settings.lantern_position.xy;
+    let lantern_pos = vec2(settings.lantern_position.x, -settings.lantern_position.y)*6.0;
 
```

Comment:
In the future probably would be better to use the camera transformation matrix, especially if we add multiple light sources, but for now I think its fine.